internal/config/config.go

```diff
@@ -32,6 +32,7 @@ const defaultReportingTimeout = time.Duration(5) * time.Second
 const defaultVerbose = false
 const defaultHttpTracing = false
 const defaultFailFast = false
+const defaultInsecureSsl = false
```

Comment:
I suggest renaming it to "insecureHTTPS`

Also https://google.github.io/styleguide/go/decisions#initialisms